### PR TITLE
integration tests pull Types and Clients from index instead of direct…

### PR DESCRIPTION
**Pull request by baxterjfinch**
Issue #110

### CODE REVIEW

File: packages/core-sdk/test/integration/licenseReadOnly.test.ts [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import chai, { expect } from "chai";
 import chaiAsPromised from "chai-as-promised";
-import { StoryClient, StoryReadOnlyConfig, ListLicenseRequest } from "../../src/index";
+import {
+  StoryClient,
+  StoryReadOnlyConfig,
+  ListLicenseRequest,
+  ReadOnlyClient,
+} from "../../src/index";
```
Comment (edisonz0718) [inline packages/core-sdk/test/integration/licenseReadOnly.test.ts:8]: i think we don't need to have /index here

File: packages/core-sdk/test/integration/transactionReadOnly.test.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import chai, { expect } from "chai";
 import chaiAsPromised from "chai-as-promised";
-import { StoryClient, StoryReadOnlyConfig } from "../../src/index";
+import {
+  StoryClient,
+  StoryReadOnlyConfig,
+  ReadOnlyClient,
+  ListTransactionRequest,
+} from "../../src/index";
```
Comment (edisonz0718) [inline packages/core-sdk/test/integration/transactionReadOnly.test.ts:8]: Same here, /index is not needed

File: packages/core-sdk/test/unit/resources/license.test.ts [hunk 3]
```diff
@@ -1,13 +1,12 @@
 import { expect } from "chai";
-import { LicenseClient } from "../../../src/resources/license";
+import { LicenseClient, AddressZero } from "../../../src";
 import { AxiosInstance } from "axios";
 import { createMock } from "../testUtils";
 import * as sinon from "sinon";
 import { CreateLicenseNftRequest, IPAssetType } from "../../../src";
```
Comment (edisonz0718) [inline packages/core-sdk/test/unit/resources/license.test.ts:6]: This line can be combined with the one on top

File: packages/core-sdk/test/integration/transactionReadOnly.test.ts (hunk 2 above)
Comment (baxterjfinch) [reply packages/core-sdk/test/integration/transactionReadOnly.test.ts:8]: :+1:

File: packages/core-sdk/test/unit/resources/license.test.ts (hunk 3 above)
Comment (baxterjfinch) [reply packages/core-sdk/test/unit/resources/license.test.ts:6]: Removed it entirely as it was unused

**Pull request merged**